src/pages/transactions/add-transaction.tsx

```diff
@@ -12,8 +15,22 @@ const categories: string[] = [
   "Tech",
 ];
 
+export const newTransactionState = atom<TransactionI>({
+  key: "newTransactionState",
+  default: {
+    type: "income",
+    concept: "",
+    category: "",
+    amount: 0,
+  },
+});
+
```

Comment:
We better keep this outside a component file, because this will be the global state for transaction